GenderPayGap.WebUI/Views/ManageOrganisations/ManageOrganisations.cshtml

```diff
@@ -0,0 +1,104 @@
+﻿@using GenderPayGap.Database
+@model GenderPayGap.WebUI.Models.Organisation.ManageOrganisationsViewModel
+
+@{
+    ViewBag.Title = "Manage your organisations - Gender pay gap service";
+    Layout = "~/Views/GovUkFrontend/GovUkFrontendLayout.cshtml";
+}
+
+@section BeforeMain {
+    @{
+        await Html.RenderPartialAsync("../Components/Navigation/AccountTabs");
+    }
+    
+    <div class="govuk-!-margin-top-3">
+        @{
+            await Html.RenderPartialAsync("../Components/CovidMessage");
+        }
+    </div>
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">
+            Select an organisation
+        </h1>
+        
+        <p class="govuk-body">
+            Use this page to access your registered organisations or to register a new organisation.
+        </p>
+        
+        <p class="govuk-body">
+            Once you have selected an organisation you can:
+        </p>
+        <ul class="govuk-list govuk-list--bullet">
+            <li>Enter gender pay gap data and save it as a draft for publication at a later date</li>
+            <li>Publish your data on the <a class="govuk-link" href="@Url.Action("Index", "Viewing")">gender pay gap service</a></li>
+            <li>Declare whether your organisation is required to report for each reporting year</li>
+        </ul>
+        
+        <table class="govuk-table">
+            <thead class="govuk-table__head">
+                <tr class="govuk-table__row">
+                    <th scope="col" class="govuk-table__header">Organisation name</th>
+                    <th scope="col" class="govuk-table__header">Registration status</th>
+                </tr>
+            </thead>
+            <tbody class="govuk-table__body">
+                @foreach (UserOrganisation userOrganisation in Model.UserOrganisations)
+                {
+                    string statusText;

```

Comment:
I'm deeply confused about how Github is showing me this diff but these appear to still be present and not be needed any more